main.py

```diff
@@ -3,22 +3,15 @@
 
 # Function to print help messages
 def printHelp():
-    print("usage: python main.py {test, run, data, train}")
```

Comment:
@koushikjoshi the entire line was not supposed to be removed. Just remove the option of `train` from the parameter list please